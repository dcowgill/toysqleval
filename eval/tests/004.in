create table test (b boolean);

insert into test values (true);
insert into test values (false);
insert into test values (true);

select * from test;
