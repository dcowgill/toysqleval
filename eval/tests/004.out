OK
OK
OK
OK
 b
-------
 true
 false
 true
